roles/setup_pgd/defaults/main.yml

```diff
@@ -113,3 +113,36 @@ supported_pg_version:
 
 supported_pgd_version:
   - 5
+
+# The following parameters must be configured in the 'pgd_commit_scopes' list
+# the values below are examples and should be adjusted to your requirements
+#  for CAMO
+#  - cs_type: 'CAMO'
+#  for GROUP_COMMIT
+#  - cs_type: 'GROUP_COMMIT'
+#  for CAMO
+#  - cs_name: 'camo_scope_1'
+#  for GROUP_COMMIT
+#  - cs_name: 'group_commit_scope_1'
+#  - parent_group: 'pgd_cluster'
+#  for CAMO
+#  - cs_origin_node_group: 'pgd_two_nodes'
+#  for GROUP_COMMIT
+#  - cs_origin_node_group: 'pgd_gc_nodes'
+#  for CAMO
+#  - member_nodes: ['edb-primary1', 'edb-primary2']
+#  for GROUP_COMMIT
+#  - member_nodes: ['edb-primary3']
+#  - default_group_cs: true
+#  for CAMO
+#  - cs_rule: "ALL ( pgd_two_nodes ) ON visible CAMO DEGRADE ON (timeout=500s) TO ASYNC"
+#  for GROUP_COMMIT
+#  - cs_rule: "ALL ( pgd_remaining_nodes ) GROUP COMMIT"
+pgd_commit_scopes:
+  - cs_name: ''
+    cs_type: ''
+    parent_group: ''
+    cs_origin_node_group: ''
+    member_nodes: ['']
+    default_group_cs: true
+    cs_rule: ""
```

Comment:
I think these comments can be cleaned up a bit, I think one group commit example and one CAMO example is sufficient without breaking down each part. to leave `pgd_commit_scopes` empty, I think it needs to be set like `pgd_commit_scopes: ""`. All of this should take place prior to the last lines of the main.yml, where the `supported_pgd_version` var should be the last in the file.